### PR TITLE
[CMS-584] Update badge contents and add pr-body-unsupported option.

### PR DESCRIPTION
**Pull request by kporras07**
### Overview
This pull request updates support level badges content and adds a new option to allow a different pr-body if support level is unsupported.

### CODE REVIEW

**Comment by kporras07** [comment]
Created as draft waiting for docs to approve the badges docs page.

File: src/Util/SupportLevel.php
Comment (greg-1-anderson) [inline src/Util/SupportLevel.php:40]: I guess we'll drop pilot again.

File: src/Cli/OrgCommands.php
Comment (greg-1-anderson) [inline src/Cli/OrgCommands.php:231]: If we don't combine unsupported and deprecated into a single support level, this conditional should probably also use the unsupported text for deprecated projects

**Review by greg-1-anderson — APPROVED** [review]

File: src/Cli/OrgCommands.php
Comment (kporras07) [reply src/Cli/OrgCommands.php]: Done

File: src/Util/SupportLevel.php
Comment (kporras07) [reply src/Util/SupportLevel.php]: Done

**Comment by kporras07** [comment]
Setting as Ready for Review/Merge and waiting for final confirmation in Slack before merging

**Pull request merged**
